process/best_practices_badge.md

```diff
@@ -0,0 +1,57 @@
+
+## "The project MUST have at least one primary developer who knows how to design secure software." - how to fulfill this?
+
+Having a primary developer take the three courses outlined below ( all free to audit ) would be sufficient:
+
+- [Secure Software Development: Requirements, Design, and Reuse (LFD104x) - Linux Foundation - Training](https://training.linuxfoundation.org/training/secure-software-development-requirements-design-and-reuse-lfd104/) 
+- [Secure Software Development: Implementation (LFD105x) - Linux Foundation - Training](https://training.linuxfoundation.org/training/secure-software-development-implementation-lfd105/) 
+- [Secure Software Development: Verification and More Specialized Topics (LFD106x) - Linux Foundation - Training](https://training.linuxfoundation.org/training/secure-software-development-verification-and-more-specialized-topics-lfd106/) 
+
+## Is there a list or document which describes the static analysis tools that ASWF supports or recommends?
+
+SonarCloud is the tool that multiple projects have implemented, and there is already an organization-level access token that projects can use with it. You can see a specific example in the OpenEXR analysis_workflow.yml, which shows how to build, run the test suite, post-process, and upload to SonarCloud using the SONAR_TOKEN from the organization level. You can see the results for the various ASWF projects with SonarCloud here:
+
+[https://sonarcloud.io/organizations/academysoftwarefoundation/projects](https://www.google.com/url?q=https://sonarcloud.io/organizations/academysoftwarefoundation/projects&sa=D&source=docs&ust=1675375752427760&usg=AOvVaw3aJ1DsMv9yF2Xf-Uhv8EKo)
+
+I'm also seeing some work towards using clang-tidy in Imath: [https://cs.github.com/AcademySoftwareFoundation/Imath/blob/3ad5d4da02795dbf37b44c5bbd2f919a7c01a19a/config/ImathSetup.cmake#L118](https://www.google.com/url?q=https://cs.github.com/AcademySoftwareFoundation/Imath/blob/3ad5d4da02795dbf37b44c5bbd2f919a7c01a19a/config/ImathSetup.cmake%23L118&sa=D&source=docs&ust=1675375752432129&usg=AOvVaw1HThXkwYas6U7fU6_IKetY)
+
+and in OSL:
+
+[https://cs.github.com/AcademySoftwareFoundation/Imath/blob/3ad5d4da02795dbf37b44c5bbd2f919a7c01a19a/config/ImathSetup.cmake#L118](https://www.google.com/url?q=https://cs.github.com/AcademySoftwareFoundation/Imath/blob/3ad5d4da02795dbf37b44c5bbd2f919a7c01a19a/config/ImathSetup.cmake%23L118&sa=D&source=docs&ust=1675375752432249&usg=AOvVaw10Qy3WPYpoaDl6XTJfp3F9)
+
+## "The project MUST have a documented process for responding to vulnerability reports." - does GitHub have anything to help with this?
+
+
+
+## "The project MUST list external dependencies in a computer-processable way." - is there an automated way to build this?
+
+LFX Security automatically will pull the list of dependencies for a project provided they use the standard mechanisms leveraged by Synk.
```

Comment:
Does that work for C++?

Reply:
Synk -> Snyk

Unfortunately Snyk does not currently support C / C++, although it does support Python which is relevant to several of our projects. The topic of C++ dependency management remains open...

https://docs.snyk.io/scan-application-code/snyk-code/snyk-code-language-and-framework-support

More work is required here.